packages/plugin-core/src/workspace/workspaceActivator.ts

```diff
@@ -548,12 +548,18 @@ export class WorkspaceActivator {
     const reloadSuccess = await reloadWorkspace({ ext, wsService });
     const durationReloadWorkspace = getDurationMilliseconds(start);
 
-    // NOTE: tracking is not awaited, don't block on this
-    ExtensionUtils.trackWorkspaceInit({
-      durationReloadWorkspace,
-      activatedSuccess: !!reloadSuccess,
-      ext,
-    });
+    // check if we have errors during init
+    try {
+      // NOTE: tracking is not awaited, don't block on this
+      ExtensionUtils.trackWorkspaceInit({
+        durationReloadWorkspace,
+        activatedSuccess: !!reloadSuccess,
+        ext,
+      });
+    } catch (error) {
```

Comment:
This won't catch anything since `ExtensionUtils.trackWorkspaceInit` is async - this needs to go in the `catch` clause of the promise.

`ExtensionUtils.trackWorkspaceInit().catch(reason => captureEx(reason));`

Reply:
nice catch!